api-reference/beta/resources/alert.md

```diff
@@ -0,0 +1,120 @@
+|applicationStates|[applicationSecurityState](applicationsecuritystate.md) collection|Security-related stateful information generated by the provider about the application(s) related to this alert. |
+|assignedTo|String|Name of the analyst the alert is assigned to for triage, investigation, or remediation ([Supports Update](../api/alert_update.md)).|
+|azureSubscriptionId|String|Azure subscription ID, present if this alert is related to an Azure resource.|
+|azureTenantId|String *|Azure Active Directory tenant ID.|
+|category|String|Category of the alert (e.g. credentialTheft, ransomware, etc.).|
+|closedDateTime|DateTimeOffset|Time at which the alert was closed. The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 would look like this: `'2014-01-01T00:00:00Z'` ([Supports Update](../api/alert_update.md)).|
+|comments|String|Customer-provided comments on alert (for customer alert management) ([Supports Update](../api/alert_update.md)).|
+|createdDateTime|DateTimeOffset *|Time at which the alert was created by the alert provider. The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 would look like this: `'2014-01-01T00:00:00Z'`.|
+|description|String|Alert description.|
+|detectionIds|String collection|Set of alerts related to this alert entity (each alert is pushed to the SIEM as a separate record).|
+|eventDateTime|DateTimeOffset *|Time at which the event(s) that served as the trigger(s) to generate the alert occurred. The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 would look like this: `'2014-01-01T00:00:00Z'`.|
+|feedback|string|Analyst feedback on the alert. Possible values are: `unknown`, `truePositive`, `falsePositive`, `benignPositive`. ([Supports Update](../api/alert_update.md))|
+|fileStates|[fileSecurityState](filesecuritystate.md) collection|Security-related stateful information generated by the provider about the file(s) related to this alert.|
+|hostStates|[hostSecurityState](hostsecuritystate.md) collection|Security-related stateful information generated by the provider about the host(s) related to this alert.|
+|id|String *|Provider-generated GUID/unique identifier. (Read-only)|
+|lastModifiedDateTime|DateTimeOffset|Time at which the alert entity was last modified. The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 would look like this: `'2014-01-01T00:00:00Z'`.|
+|malwareStates|[malwareState](malwarestate.md) collection|Threat Intelligence pertaining to malware related to this alert.|
+|malwareWasRunning|Boolean|Was the detected file (malware/vulnerability) running at the time of detection or was it detected at rest on disk?|

```

Comment:
Why is this in interrogative form? Change to "Indicates whether the detected file (malware/vulnerability) was running at the time of detection or was detected at rest on disk."